src/pool/pool_scalable.c

```diff
@@ -109,42 +111,46 @@ static const char *tbb_symbol[TBB_POOL_SYMBOLS_MAX] = {
 #endif
 };
 
-static int init_tbb_callbacks(tbb_callbacks_t *tbb_callbacks) {
-    assert(tbb_callbacks);
-
+static int init_tbb_callbacks() {
```

Comment:
`void` should be used here (`static int init_tbb_callbacks(void) {`) - there are compilers that require that (on MacOS AFAIR)